src/main/java/games/strategy/util/Version.java

```diff
@@ -15,7 +15,7 @@
   private final int m_major;
   private final int m_minor;
   private final int m_point;
-  private final int m_micro;
+  private final int build;
```

Comment:
We are not yet using the version proxy, would renaming this break serialization compatibility of save games? Are we assuming this update for the next incompatible release?

Reply:
It is also odd to be inconsistent within a single class file w.r.t naming. Either stick with 'm_' for this one new variable, or more ideally update them all (and even more ideally as a preliminary branch so that can be reviewed on its own, but perhaps a bit late now).

Reply:
Yes, this will break compatibility.  `Version` is serialized with `GameData`.

Reply:
Right. Is there an easy way of removing a commit?

Reply:
https://git-scm.com/docs/git-revert

Reply:
`git rebase -i HEAD~n`, where 'n' is the number of commits you want to go back. That shows you a script screen where the first column is a 'command' to git to squash, delete or edit commits. If you delete the commit line then it should be removed from your branch. You will though need to force push the update to rewrite this branch. It may be easiest instead to do a `git revert <commit SHA>` to back out the update.